app/plex_tools.py

```diff
@@ -178,43 +206,72 @@ def add_to_collection(config_path, plex, method, value, c, map, subfilters=None)
         for rk in movies:
             current_m = get_movie(plex, rk)
             current_m.reload()
-            if current_m in fs:
-                print("| {} Collection | = | {}".format(c, current_m.title))
-                map[current_m.ratingKey] = None
-            elif subfilters:
-                match = True
+            match = True
+            if subfilters:
                 for sf in subfilters:
-                    method = sf[0]
-                    terms = str(sf[1]).split(", ")
-                    try:
-                        mv_attrs = getattr(current_m, method)
-                        # If it returns a list, get the 'tag' attribute
-                        # Otherwise, it's a string. Make it a list.
-                        if isinstance(mv_attrs, list) and "-" not in method:
-                            mv_attrs = [getattr(x, 'tag') for x in mv_attrs]
-                        else:
-                            mv_attrs = [str(mv_attrs)]
-                    except AttributeError:
-                        for media in current_m.media:
-                            if method == "video_resolution":
-                                mv_attrs = [media.videoResolution]
-                            for part in media.parts:
-                                if method == "audio_language":
-                                    mv_attrs = ([audio_stream.language for audio_stream in part.audioStreams()])
-                                if method == "subtitle_language":
-                                    mv_attrs = ([subtitle_stream.language for subtitle_stream in part.subtitleStreams()])
+                    method = alias(sf[0], subfilter_alias)
+                    negative = False
+                    gte = False
+                    lte = False
+                    if method[-1] == "!":
+                        negative = True
+                        method = method[:-1]
+                    elif method[-4] == ".lte":
+                        lte = True
+                        method = method[:-4]
+                    elif method[-4] == ".gte":
+                        gte = True
+                        method = method[:-4]
+                    if method == "days_from_now":
```

Comment:
Can we call this `max-age`, with days being the unit? Even nicer (but not necessary), support values like `30d` or `1y`?

Reply:
absolutely lol `max_age` is way better than `days_from_now` and ill have it default to days with a straight int but if it ends with the `y` we can do years

Reply:
I say keep it simple... just support days